scripts/simulate_coordinator_upgrade.py

```diff
@@ -0,0 +1,34 @@
+# Usage:
+#  > ape run --interactive simulate_coordinator_upgrade --network polygon:mainnet-fork:foundry
+
+from ape import accounts, chain, project
+
+from deployment.constants import CONSTRUCTOR_PARAMS_DIR
+from deployment.params import Deployer
+
+
+OZ_DEPENDENCY = project.dependencies["openzeppelin"]["5.0.0"]
+PROXY_ADMIN_ADDRESS = "0xeE711368eabA106A0cf7a07B33B84cD930331fFd"
+COORDINATOR_PROXY_ADDRESS = "0xE74259e3dafe30bAA8700238e324b47aC98FE755"
+CONSTRUCTOR_PARAMS_FILEPATH = CONSTRUCTOR_PARAMS_DIR / "mainnet" / "redeploy-coordinator.yml"
+NUCO_MULTISIG_ADDRESS = "0x861aa915C785dEe04684444560fC7A2AB43a1543"
+
+
+def main():
+    nuco_multisig = accounts.test_accounts.impersonate_account(NUCO_MULTISIG_ADDRESS)
```

Comment:
You could add the following to validate the script is being run on `mainnet-fork` (needs a `networks` import above if you decide to use it)

```suggestion
def main():
    if networks.active_provider.network.name != "mainnet-fork":
        raise RuntimeError("This script must be run on mainnet-fork only!")
        
    nuco_multisig = accounts.test_accounts.impersonate_account(NUCO_MULTISIG_ADDRESS)
```